CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Onboarding/OnboardingDataSource.swift

```diff
@@ -89,10 +85,9 @@ class OnboardingDataSource: NSObject, BaseCollectionViewDataSource {
 
     private func configureCell(cell: OnboardingPageCell, indexPath: IndexPath) {
         let item = itemSections[indexPath.row]
-        
+       
         if let predefinedPageType = OnboadingPageType.init(rawValue: indexPath.row),
-            let adapter = adapters[predefinedPageType],
-            adapter != nil {
+            let adapter = adapters[predefinedPageType], adapter != nil {
```

Comment:
```suggestion
            let adapter = adapters[predefinedPageType] {
```

Reply:
@ValentinaIancu-Gini,  do we still need to check the adapter for nil?

Reply:
nop, I forgot to remove it ;). Thanks! I will remove it.